lib/msal-browser/docs/configuration.md

```diff
@@ -1,2 +1,81 @@
 # Configuration Options
 
+Before you start here, make sure you understand how to [initialize an app object](./initialization.md).
+
+The MSAL library has a set of configuration options that can be used to customize the behavior of your authentication flows. These options can be set either in the constructor of the `PublicClientApplication` object or as part of the [request APIs](./request-response-object.md). Here we describe the configuration object that can be passed into the `PublicClientApplication` constructor. 
+
+## Using the config object
+
+The configuration object has the following structure, and can be passed into the `PublicClientApplication` constructor. The only required config parameter is the client ID of the application. Everything else is optional, but may be required depending on your tenant and application model.
+
+```javascript
+const msalConfig = {
+    auth: {
+        clientId: "enter_client_id_here",
+        authority: "https://login.microsoftonline.com/common",
+        redirectUri: "enter_redirect_uri_here",
+        postLogoutRedirectUri: "enter_postlogout_uri_here",
+        navigateToLoginRequestUrl: true
+    },
+    cache: {
+        cacheLocation: "sessionStorage",
+        storeAuthStateInCookie: false
+    },
+    system: {
+        loggerOptions: {
+            loggerCallback: (level: LogLevel, message: string, containsPii: boolean): void => {
```

Comment:
Haven't we decided to have the option for the library to work without the logger call back with "debug"?

Reply:
We can remove this when the logger changes go in, for now I am going to provide a callback for logs.